interface.html

```diff
@@ -59,6 +59,36 @@
     </div>
   </div>
 
+  <h3></h3>
+  <div class="form-group clearfix">
+    <div class="col-sm-4 control-label">
+      <label for="loggedInUserTime">Keep the user logged in for</label>
+      <a class="help-icon" data-toggle="tooltip" data-placement="top" title="Force user re-authentication after a number of hours">
+        <i class="fa fa-question-circle-o"></i>
+      </a>
+    </div>
+    <div class="col-sm-8">
+      <div class="input-group">
+        <input type="number" class="form-control logged-user" name="loggedInUserTime" min="0" oninput="validity.valid||(value='');" value="{{sessionMaxDurationMinutes}}">
+        <span class="input-group-addon time-label">Hours</span>
+      </div>
+    </div>
+  </div>
+  <div class="form-group clearfix">
+    <div class="col-sm-4 control-label">
+      <span for="loggedOutUserTime" class="logged-out-user-time">Automatically log out the user due to inactivity after</span>
```

Comment:
Why is this not using `label` which comes with the standard bold style for Bootstrap?

```suggestion
      <label for="loggedOutUserTime">Automatically log out the user due to inactivity after</label>
```

Reply:
@tonytlwu your right. but due the large text help-icon goes down when we are using `<label>`. I also tried with different way like used  `<p>` , pull-left, pull-right classes etc but not working properly. So I used `<span>` here.
Also Please see below screen recording:

https://user-images.githubusercontent.com/102005879/194691783-a9d030f9-fb65-4289-8d6d-feb0d0c179ff.mp4

Reply:
That's because the `label` is a block element. Put the `a` inside the `label` tag at the end then it should be fine.